_data/security.yml

```diff
@@ -179,6 +179,12 @@ websites:
       - u2f
     doc: https://support.norton.com/sp/en/us/home/current/solutions/v100023155_NortonM_Retail_1_en_us
 
+  - name: Panther
```

Comment:
```suggestion
  - name: Panther Labs
```

Reply:
I had it at "Panther Labs" initially, but since the website is called "runpanther.io", I suppose it's more focused on the product rather than the company.

Reply:
Yeah that makes sense. :+1: